diff --git a/doc_src/fish_indent_lexer.py b/doc_src/fish_indent_lexer.py
index f4b154e8c..530856a5d 100644
--- a/doc_src/fish_indent_lexer.py
+++ b/doc_src/fish_indent_lexer.py
@@ -91,11 +91,12 @@ def tokenize_fish_command(code, offset):
     stdout, _ = proc.communicate(code.encode("utf-8"))
     result = []
     for line in stdout.decode("utf-8").splitlines():
-        start, end, role = line.split(",")
-        start, end = int(start), int(end)
-        value = code[start:end]
-        tok = token_for_text_and_role(value, role)
-        result.append((start + offset, tok, value))
+        if line:
+            start, end, role = line.split(",")
+            start, end = int(start), int(end)
+            value = code[start:end]
+            tok = token_for_text_and_role(value, role)
+            result.append((start + offset, tok, value))
     return result
 
 
